### ISSUE TITLE
[Bug] Reports: adding multiple proteins to comma separated file

### ISSUE THREAD

**Issue by tivdnbos** [compomics/peptide-shaker#440]
When multiple proteins are added to a report, a comma is used. This occurs even when the user prefers csv format instead of tab. Therefore, no major problems here: the user can use the default tab separated format instead of comma separated, but I thought you should be aware of this.
This problem might also occur for other features, but I haven't tested this.

**Comment by hbarsnes** [comment]
Good point! Any suggestions for what we ought to use instead of comma? As we still want the text to be easy to read.

**Comment by tivdnbos** [comment]
Sorry for my late reply, Harald. 

I think the most convenient way is to _not allow_ csv files, but only the default tsv files so no parsing errors can occur. Perhaps, a semicolon might be of use? 

Best,
Tim

**Comment by hbarsnes** [comment]
Hi Tim,

Now that you mentioned it, aren't all our default text exports tab based, i.e. tsv files and not csv files? Which means that this issue only occurs if a user chooses to use comma separated columns and selecting export content such as protein groups where comma is also used to separate the column content? So the easy fix is simply to not create those types of reports I guess? ;)

I'm leaning against not removing csv as an output format though, as there might be simpler user-defined exports that do not include comma-separated column content.

And from a general point of view, there is really no symbol that can be considered as "safe" when it comes to protein names and accession numbers, as I do not think that we can guarantee that there will be no protein names or accession numbers containing comma or semi colon, for example.

Best regards,
Harald
